tests/Unit/Parallel/Test_ConstGlobalCache.cpp

```diff
@@ -88,14 +94,16 @@ class Square : public Shape {
 
 struct shape_of_nametag_base {};
 
-struct shape_of_nametag : shape_of_nametag_base {
+struct shape_of_nametag : shape_of_nametag_base, db::SimpleTag {
```

Comment:
If you're going to fix the names on some of the classes in this file you might as well do them all.

Reply:
I only had to fix the `name`->`Name` tag to avoid collision with its `name()` function. But sure, I can change the other names, too.